models/br_anp_precos_combustiveis/br_anp_precos_combustiveis__microdados.sql

```diff
@@ -11,9 +11,18 @@
         "interval": 1}
     },
     cluster_by = ["id_municipio", "sigla_uf"],
-    labels = {'project_id': 'basedosdados'})
-}}
-
+    labels = {'project_id': 'basedosdados-dev'},
+    pre_hook = "DROP ALL ROW ACCESS POLICIES ON {{ this }}",
```

Comment:
Vai crashar a execução do modelo DBT se settar o  pre_hook = "DROP ALL ROW ACCESS POLICIES ON {{ this }}".  Ele vai tentar tirar Policies que não existem e vai crashar. Primeiro mergeia um PR com o post_hook e executa o modelo para criar a policy. Em seguida cria outro PR com pre_hook.